lib/parse.js

```diff
@@ -120,45 +143,59 @@ function parsePlistXML (node) {
     // TODO: what should we do with text types? (CDATA sections)
 
   } else if (node.nodeName === 'key') {
-    if(isEmptyNode(node)) return null;
     res = '';
-    if(node.childNodes[0]) {
+    if(!isEmptyNode(node)) {
       res = node.childNodes[0].nodeValue;
```

Comment:
another early return opportunity. I think you can just `return node.childNodes[0].nodeValue` rather than filling a variable and returning that?